inst/rmarkdown/templates/flanders_slides/skeleton/skeleton.Rmd

```diff
@@ -8,7 +8,8 @@ output:
     location: "The location and date"
     institute: "Affilitation of the author(s)"
     theme: "vlaanderen"
-    toc_name: "Summmary"
+    flandersfont: TRUE
```

Comment:
OK, done in commit https://github.com/inbo/INBOmd/pull/18/commits/31bef653c911df4f74f1251e806951c2cbd8d4b9